### ISSUE TITLE
R&D implementation of "identities" related data scopes

### ISSUE THREAD

**Issue by paulobmarcos** [ipfs-shipyard/pm-idm#108]
## Description

Research how all the `identities` related data scopes will be implemented in the IDM Wallet module according to the IDM spec.

## Acceptance Criteria

- [x] Analyse Orbit DB Store Packages and its usages by each scope needs
- [ ] ~~Discuss the creation of a new module to manage credentials~~

**Comment by satazor** [comment]
Related to #88

**Comment by satazor** [comment]
@paulobmarcos can you summarize the R&D and the tradeoffs we decided to do for now?
